### ISSUE TITLE
gftools-add-font.py: ParseError for Variable Font

### ISSUE THREAD

**Issue by yanone** [googlefonts/gftools#168]
Adding the Petrona family throws a ParseError on [this font](https://github.com/ringorseeber/Petrona/blob/master/Fonts/vf/Petrona%5Bwght%5D.ttf).
(Font Bakery checks pass fine)

```
Traceback (most recent call last):
  File "/usr/local/bin/gftools-add-font.py", line 269, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/usr/local/bin/gftools-add-font.py", line 255, in main
    metadata = _MakeMetadata(fontdir, is_new)
  File "/usr/local/bin/gftools-add-font.py", line 116, in _MakeMetadata
    file_family_style_weights = _FileFamilyStyleWeights(fontdir)
  File "/usr/local/bin/gftools-add-font.py", line 91, in _FileFamilyStyleWeights
    result = [fonts.FileFamilyStyleWeight(f) for f in files]
  File "/usr/local/bin/gftools-add-font.py", line 91, in <listcomp>
    result = [fonts.FileFamilyStyleWeight(f) for f in files]
  File "/usr/local/lib/python3.7/site-packages/gftools/util/google_fonts.py", line 463, in FileFamilyStyleWeight
    raise ParseError('Could not parse %s' % filename)
gftools.util.google_fonts.ParseError: Could not parse /Users/yanone/Projekte/Google/Onboarding/fonts/ofl/petrona/Petrona[wght].ttf
```

**Comment by m4rc1e** [comment]
Today I merged #154 which should make add-font work on VFs. Please reinstall your gftools by doing

`pip install git+https://www.github.com/googlefonts/tools --upgrade`

**Comment by yanone** [comment]
Awesome, it worked. Thanks.
